rails-intro.md

```diff
@@ -0,0 +1,94 @@
+# How do websites work?
+
+At the most basic level Rails is a tool for listening to HTTP requests from user's web browers and return HTTP responses. When you visit any website on the internet your browser constructs an HTTP request and sends it to the server that is running that website. Every website on the internet has a piece of software running that listens for those HTTP requests and prepares an HTTP response to send back to your browser.
```

Comment:
A paradigm that has worked well for me is the idea of questions / answers.

Clients ask well structured questions. Servers answer them.

I couldn't think of a way to weave that in here. So just leaving as a comment in case it sparks someone's brain.

Reply:
I think this is good. Having an analogy running through this would help anchor some of the abstract concepts. I tacked it on to the end of this paragraph!

Reply:
And then I used that analogy in the explanation of the request methods as well!